roles/confluent.variables/defaults/main.yml

```diff
@@ -133,10 +133,10 @@ ssl_custom_certs_remote_src: false
 
 # Zookeeper Variables
 
-### Only use to customize Linux User Zookeeper Service runs with. User must exist on host.
+### Only use to customize Linux User Zookeeper Service runs with. Default user is cp-kafka
 zookeeper_user: "{{zookeeper_default_user}}"
 
-### Only use to customize Linux Group Zookeeper Service user belongs to. Group must exist on host.
+### Only use to customize Linux Group Zookeeper Service user belongs to. Default group is confluent
```

Comment:
Modify this variable to customize the Linux Group that the Zookeeper Service user belongs to. Default group is confluent.